ati_wrench_publisher:
  ros__parameters:
    frame_id: "end_effector_link"
    pub_faked_wrench: True
    ip_address: "192.168.1.84"
